swarm/AUTHORS

```diff
@@ -0,0 +1,35 @@
+# Core team members
+
+Viktor Trón - @zelig
+Louis Holbrook - @nolash
+Lewis Marshal - @lmars
+Anton Evangelatov - @nonsense
+Janos Gulyas - @janos
```

Comment:
My name is better to be spelled as it is on github page "Janoš Guljaš". Thanks.

Reply:
Fixed in 55c2bc2eb5603616b04f891d74e88382a288354a